src/dag-node/create.js

```diff
@@ -9,36 +9,25 @@ const linkSort = dagNodeUtil.linkSort
 const DAGNode = require('./index.js')
 const DAGLink = require('../dag-link')
 
-function create (data, links, callback) {
-  if (typeof data === 'function') {
-    callback = data
-    data = undefined
-  } else if (typeof data === 'string') {
+const create = async (data, links = []) => {
```

Comment:
`async` can be removed from this because there's only one `await`, which itself is unnecessary and should be removed